apps/indexer/config/dev/geth.exs

```diff
@@ -6,7 +6,8 @@ config :indexer,
     transport: EthereumJSONRPC.HTTP,
     transport_options: [
       http: EthereumJSONRPC.HTTP.HTTPoison,
-      url: System.get_env("ETHEREUM_JSONRPC_HTTP_URL") || "https://mainnet.infura.io/8lTvJTKmHPCHazkneJsY",
+      url:
+        System.get_env("ETHEREUM_JSONRPC_HTTP_URL") || "https://mainnet.infura.io/v3/8a14134acf5d4d7ababe7a94484bbffb",
```

Comment:
@dviramontes we shouldn't put our infura access code in the source code 😉

Reply:
its not ours its mine personal

Reply:
reverted

Reply:
word